tests/regression/pip_faster.py

```diff
@@ -38,77 +38,19 @@ def test_circular_dependencies():
         '-vv',  # show debug logging
         'circular-dep-a',
     )
-    assert err == ''
+    assert err == 'Circular dependency! circular-dep-a==1.0 (from circular-dep-b==1.0->circular-dep-a)\n'
     out = uncolor(out)
     assert out.endswith('''
 tracing: circular-dep-a
 already queued: circular-dep-b==1.0 (from circular-dep-a)
 tracing: circular-dep-b==1.0 (from circular-dep-a)
-Circular dependency! circular-dep-a==1.0 (from circular-dep-b==1.0->circular-dep-a)
 ''')
 
     frozen_requirements = pip_freeze(str(venv)).split('\n')
     assert 'circular-dep-a==1.0' in frozen_requirements
     assert 'circular-dep-b==1.0' in frozen_requirements
 
 
-@pytest.mark.usefixtures('pypi_server')
-@pytest.mark.skipif(
-    sys.version_info > (3, 0),
-    reason='ancient versions are not py3 compatible, even for install',
-)
-@pytest.mark.parametrize('reqs', [
-    # old setuptools and old pip
-    ['setuptools==0.6c11', 'pip==1.4.1'],
-    # old setuptools and new pip
-    ['setuptools==0.6c11', 'pip==1.5.6'],
-    # new setuptools and old pip
-    ['setuptools==18.2', 'pip==1.4.1'],
-])
-def test_old_pip_and_setuptools(tmpdir, reqs):
```

Comment:
You misunderstand the test I think. The point is that venv-update works even starting from an environment with old pip. That continues to be true even if we start requiring newer pip; venv-update will/should upgrade it.